llvmlite/ir/values.py

```diff
@@ -14,13 +14,15 @@
 
 _VALID_CHARS = (frozenset(map(ord, string.ascii_letters)) |
                 frozenset(map(ord, string.digits)) |
-                frozenset('._-$'))
+                frozenset(map(ord, ' !#$%&\'()*+,-./:;<=>?@[]^_`{|}~')))
```

Comment:
Well, I guess there's this branch below...
```

    if six.PY2 and isinstance(text, bytes):
        buf = [_map[ord(ch)] for ch in text]
    else:
        buf = [_map[ch] for ch in text]
```
so using `ord()` would pessimize py3. This is a rather hot function since IR contains many strings, and I don't think that should be done.